src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -415,7 +444,38 @@ func (d *Dumper) getPodSummary(resource, podName, crName string, namespace strin
 		ports = port + ":5432"
 		summCmdName = "sh"
 		summCmdArgs = []string{"-c", "curl https://raw.githubusercontent.com/percona/support-snippets/master/postgresql/pg_gather/gather.sql" +
-			" 2>/dev/null | PGPASSWORD=" + string(pass) + " psql -X --host=127.0.0.1 --port=" + port + " --user=" + user}
+			" 2>/dev/null | PGPASSWORD='" + string(pass) + "' psql -X --host=127.0.0.1 --port=" + port + " --user='" + user + "'"}
+	case "pgv2":
+		var user, pass, port string
+		if d.forwardport != "" {
+			port = d.forwardport
+		} else {
+			port = "5432"
+		}
+		cr, err := d.getCR("perconapgclusters", namespace)
+		if err != nil {
+			return nil, errors.Wrap(err, "get cr")
+		}
+		if cr.Spec.SecretName != "" {
+			user, err = d.getDataFromSecret(cr.Spec.SecretName, "username", namespace)
+		} else {
+			user, err = d.getDataFromSecret(crName+"-pguser-cluster1", "user", namespace)
```

Comment:
the secret name depends on the cluster name

Reply:
So it should be something like
```suggestion
			user, err = d.getDataFromSecret(crName+"-pguser-"+crName, "user", namespace)
```
?